### PR TITLE
Add xtensor-r

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipes/xtensor-r/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D BUILD_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
```
Comment (ocefpaf) [inline recipes/xtensor-r/bld.bat]: Is there a `ctest`?

File: recipes/xtensor-r/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+make install
```
Comment (ocefpaf) [inline recipes/xtensor-r/build.sh]: `make check`?

File: recipes/xtensor-r/bld.bat (hunk 1 above)
Comment (SylvainCorlay) [reply recipes/xtensor-r/bld.bat]: Yes, but xtensor, xtensor-r, xtensor-foobar all require C++14 to run the tests.

I will be running the tests as part of the recipes when the switch to gcc7 is done.

File: recipes/xtensor-r/build.sh (hunk 2 above)
Comment (SylvainCorlay) [reply recipes/xtensor-r/build.sh]: same.

**Comment by SylvainCorlay** [comment]
Thanks!

**Comment by ocefpaf** [comment]
> Thanks!

No problem. Is the testing situation the same in the other two PRs you requested review?

**Pull request merged**